src/detect-engine-analyzer.c

```diff
@@ -861,6 +863,44 @@ static void DumpMatches(RuleAnalyzer *ctx, JsonBuilder *js, const SigMatchData *
                 jb_close(js);
                 break;
             }
+            case DETECT_FLOWBITS: {
+                const DetectFlowbitsData *cd = (const DetectFlowbitsData *)smd->ctx;
+
+                jb_open_object(js, "flowbits");
+                switch (cd->cmd) {
+                    case DETECT_FLOWBITS_CMD_NOALERT:
+                        jb_set_string(js, "action", "noalert");
+                        break;
+                    case DETECT_FLOWBITS_CMD_ISSET:
+                        jb_set_string(js, "cmd", "isset");
+                        break;
+                    case DETECT_FLOWBITS_CMD_ISNOTSET:
+                        jb_set_string(js, "cmd", "isnotset");
+                        break;
+                    case DETECT_FLOWBITS_CMD_SET:
+                        jb_set_string(js, "cmd", "set");
+                        break;
+                    case DETECT_FLOWBITS_CMD_UNSET:
+                        jb_set_string(js, "cmd", "unset");
+                        break;
+                    case DETECT_FLOWBITS_CMD_TOGGLE:
+                        jb_set_string(js, "cmd", "toggle");
+                        break;
+                }
+                if (cd->or_list_size == 0) {
+                    jb_set_string(js, "name", VarNameStoreSetupLookup(cd->idx, VAR_TYPE_FLOW_BIT));
+                } else if (cd->or_list_size > 0) {
+                    jb_set_uint(js, "ored_flowbits", cd->or_list_size);
+                    jb_open_array(js, "ored_variables");
```

Comment:
I think we should keep the output simple. Like
```
flowbits:
   name: fb1
```
or
```
flowbits:
  name: [fb1, fb2]  # in case of an or operation
```

Also, one thing to note is that even in an or list, not all flowbits may have matched. From what I gather, we want to log the postmatch stuff so we should only log the names of the flowbit that matched? I'll check today and get back if there's a better way to retrieve it.

Also, need to look if or list may be specific to certain cmds.

Reply:
I'm not sure I understand what you mean by changing the output. Currently, I have output being stored in `name` and the ored variables in `ored_flowbits`. Should I just have the `name` output? Also, or list is specific only to isset and isnotset cmds from what I understand.

Reply:
There's no such thing as ored variables. Those are also flowbit names separated by `|` which signifies `or` operator. And, there are flowbit names which are individually present in a rule.
I still have to check the code a bit so please wait on it. I'll get to it very soon now.